olp-cpp-sdk-dataservice-write/src/StreamLayerClientImpl.cpp

```diff
@@ -90,7 +90,7 @@ StreamLayerClientImpl::StreamLayerClientImpl(HRN catalog,
       cache_(settings_.cache),
       cache_mutex_(),
       flush_settings_(std::move(flush_settings)),
-      auto_flush_controller_(new AutoFlushController(flush_settings_)) {}
+      auto_flush_controller_(new AutoFlushController(AutoFlushSettings{})) {}
```

Comment:
pass the settings?

Reply:
You mean to pass `flush_settings_`? Actually, it is not needed, because auto-flush is disabled. I will remove the `auto_flush_controller` from this class in next PRs